src/box/lua/iproto.c

```diff
@@ -287,23 +168,38 @@ push_iproto_protocol_features(struct lua_State *L)
 	lua_pushinteger(L, IPROTO_CURRENT_VERSION);
 	lua_setfield(L, -2, "protocol_version");
 
-	const struct iproto_constant features[] = {
-		{"streams", IPROTO_FEATURE_STREAMS},
-        	{"transactions", IPROTO_FEATURE_TRANSACTIONS},
-        	{"error_extension", IPROTO_FEATURE_ERROR_EXTENSION},
-        	{"watchers", IPROTO_FEATURE_WATCHERS},
-        	{"pagination", IPROTO_FEATURE_PAGINATION},
-	};
+	size_t region_svp = region_used(&fiber()->gc);
+	char **names = xregion_alloc_array(
+		&fiber()->gc, char *,
+		iproto_feature_id_constants_size
+	);
```

Comment:
I would prefer casting `features[i].name`, but up to second reviewer.

Reply:
We decided to inline `push_iproto_constant_subnamespace` here - semantics of stripped prefix isn't used, `strstr("RESERVED")` is not actual too since features are in lowercase. So the actually used part is trivial.